### PR TITLE
Updating linux to allow for more installation methods

### LINKED ISSUE #36: Linux 15.04 Fails installation on tutorial
Is docs.docker.com/linux/step_one/ up to date? Which Linux distro are you testing with? "docker run ..." step doesn't work with Ubuntu 15.04

Most likely need to update to rpm installation with apt.

### PR DESCRIPTION
**Pull request by moxiegirl**
Closes #36 
Signed-off-by: Mary Anthony <mary@docker.com>

### CODE REVIEW

**Comment by moxiegirl** [comment]
@thaJeztah @SvenDowideit 

author LGTM

File: docs/linux/step_one.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ weight = 1
 
 # Install Docker
 
+This installation procedure is written for users who are unfamiliar with package
+managers. If you comfortable with package managers, prefer not to use `wget`, or
+have problems installing and want to troubleshoot, please use our `apt` and
+`yum` <a href="http://docs.docker.com/engine/installation/"
```
Comment (thaJeztah) [inline docs/linux/step_one.md]: can you make this https:// ?

File: docs/linux/step_one.md [hunk 2]
```diff
@@ -75,7 +81,7 @@ weight = 1
 
 At this point, you have successfully installed Docker. Leave the terminal
 window open. Now, go to the next page to [read a very short introduction Docker
-images and containers](step_two.md).
+images and containers](/linux/step_two).
```
Comment (thaJeztah) [inline docs/linux/step_one.md]: is this needed? I used the old link, and it seems to work ok (`make docs`)

**Comment by thaJeztah** [comment]
LGTM, thanks!

**Pull request merged**